ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/utils/StringUtils.java

```diff
@@ -164,17 +164,12 @@ public static String getErrorStringValue(Throwable error) {
     }
 
     /**
-     * Convert unicode character to ballerina unicode format
+     * Convert unicode character to ballerina unicode format.
      *
      * @param string Input unicode containing string to convert.
```

Comment:
Update doc comment

Reply:
Updated in bd20d609da6b7d469096e76b8fad2d9ccdeb2cf9